server/tests/mochData/socialMockData.js

```diff
@@ -0,0 +1,41 @@
+const mockData = [
```

Comment:
So, what’s the difference between **socialMock** and **socialMockData**? You should use a single efficient approach between them.

Reply:
Feedback Implemented! FIle deleted